src/python/gudhi/point_cloud/timedelay.py

```diff
@@ -0,0 +1,94 @@
+# This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+# See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+# Author(s):       Martin Royer, Yuichi Ike, Masatoshi Takenouchi
+#
+# Copyright (C) 2020 Inria, Copyright (C) 2020 Fujitsu Laboratories Ltd.
+# Modification(s):
+#   - YYYY/MM Author: Description of the modification
+
+import numpy as np
+
+class TimeDelayEmbedding:
+    """Point cloud transformation class.
+    Embeds time-series data in the R^d according to Takens' Embedding Theorem
```

Comment:
Maybe add a link to wikipedia:
```suggestion
    Embeds time-series data in the R^d according to [Takens' Embedding Theorem](https://en.wikipedia.org/wiki/Takens%27s_theorem)
```

Reply:
I believe that once you add this link asked by Vincent, we are good to go :+1:

Reply:
Thank you very much.  
Now reflected.

Reply:
>  **Go please wait.**

Ok, I'll wait, let me know when I can proceed.

Reply:
**Travis CI build is failing, don't you care about this?**
https://travis-ci.org/GUDHI/gudhi-devel/builds/654797009?utm_source=github_status&utm_medium=notification

Reply:
The test that fails is about building the C++ documentation, something that your branch does not touch. It already affects master. It is an issue with installing doxygen using homebrew on macosx (I believe it appeared after travis or homebrew updated something), Vincent is trying to fix it in some other branch. It doesn't need to block your branch.

Reply:
Thank you.
I understand well.

Reply:
So, is it ok with you if I merge your branch, or do you still want me to wait?

Reply:
I want you to wait.
I'm asking my boss when to merge.

Reply:
I'm sorry my reply was late.
Please merge on **March 16 AM**.